apps/schematic/api/README.md

```diff
@@ -11,39 +11,41 @@ This example uses the [Connexion](https://github.com/zalando/connexion) library
 Python 3.5.2+
 
 ## Usage
-To run the server, please execute the following from the root directory:
-
+To run the server, please execute the following from folder `apps/schematic/api`:
 ```
-pip3 install -r requirements.txt
-python3 -m schematic_api
+poetry shell
 ```
-
-and open your browser to here:
+To install dependencies: 
 
 ```
-http://localhost:8080/v1/ui/
+poetry install 
 ```
-
-Your OpenAPI definition lives here:
+And run schematic APIs: 
 
 ```
-http://localhost:8080/v1/openapi.json
+python3 -m schematic-api
 ```
+and open your browser to here:
 
-To launch the integration tests, use tox:
 ```
-sudo pip install tox
-tox
+http://127.0.0.1:7080/api/v1/ui/
 ```
 
 ## Running with Docker
```

Comment:
@tschaffter I updated the documentation in the README.md like we discussed today. Does this look good? The nx command should be sufficient to build the docker image and run the container

Reply:
The instructions should be even more simple using the Nx API. Developers should run Nx tasks with `nx <task> <project>` or `nx run <project>:<task>` with non-nx commands being exceptions.

1. Prepare the development environment of the project with `nx prepare schematic-api`. This will create a venv and install all the Python dependencies. We could even extend this task to run your setup script automatically.

2. Start the REST API with `nx serve schematic-api`

3. Build the image with `nx build-image schematic-api`

4. Start the containerized REST API with `nx serve-detach schematic-api` (the image must be built first)

Reply:
Thanks Thomas. I updated the documentation and created two sessions: "running schematic APIs with Docker" and "running schematic APIs without Docker" . Can I also remove the overview related to "OpenAPI generated server" ? I image developers will just read the documentation to figure out running schematic APIs? And the documentation related to "OpenAPI generated server" won't be as useful?

Reply:
Yes